src/probnum/diffeq/odefiltsmooth/odefiltsmooth.py

```diff
@@ -362,31 +362,31 @@ def _string2matern(ivp, which_prior, **kwargs):
     else:
         lengthscale = 1.0
     if which_prior == "matern32":
-        return pnfs.statespace.Matern(
+        return pnss.Matern(
             1,
             ivp.dimension,
             lengthscale,
             forward_implementation="sqrt",
             backward_implementation="sqrt",
         )
     elif which_prior == "matern52":
-        return pnfs.statespace.Matern(
+        return pnss.Matern(
             2,
             ivp.dimension,
             lengthscale,
             forward_implementation="sqrt",
             backward_implementation="sqrt",
         )
     elif which_prior == "matern72":
-        return pnfs.statespace.Matern(
+        return pnss.Matern(
             3,
             ivp.dimension,
             lengthscale,
             forward_implementation="sqrt",
             backward_implementation="sqrt",
         )
     elif which_prior == "matern92":
-        return pnfs.statespace.Matern(
+        return pnss.Matern(
```

Comment:
I know this is annoying now, and that it is not part of your PR, but this line (and the related ones above) are not covered by tests (for some reason, codecov thinks this is your fault) which leads the the diff coverage being <90% (which is why you wont be able to merge I guess). What do you think about adding a quick test for these settings in ``test_odefiltsmooth.TestLotkaVolterraOtherPriors``? Or, alternatively, remove these options (as they've never been used anyway -- in the future, we will have to consider how to handle the order of the prior in a separate issue; see #272.

Reply:
I will add a test for that and then re-request you as a reviewer 👍